### PR TITLE
16 register business

### PR DESCRIPTION
**Pull request by leonardk1**
#16
As An Admin Contact
I Want To register the Business
So That I can be able to perform other business related tasks

##### Acceptance Criteria
- [x] GIVEN I Successfully Complete signing Up THEN I Should be given a form to register the Business
- [x] GIVEN I have Submitted the Business Details, THEN I should be be given notifications upon successful or non successful Submission

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

File: app/controllers/business_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def new
+    @business = Business.new
+  end
+
+  def create
+    @business = Business.new(business_params)
+    respond_to do |format|
+      if @business.save
+        flash[:success] = "Business, Successfully Added"
+        format.html { redirect_to root_path }
```
Comment (godfreym) [inline app/controllers/business_controller.rb]: @leonardk1 Upon successfully creating the new business why is the taken to root_path and why not showing him the business details he created?

**Comment by leonardk1** [comment]
@godfreym the changes have been made, please review
@arafatm please review

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -7,4 +7,12 @@ def flash_class(level)
     when :alert then "alert alert-dismissable alert-warning"
     end
   end
+
+  def attribute_present_or_not?(attribute)
+  	if attribute.present?
```
Comment (godfreym) [inline app/helpers/application_helper.rb]: @leonardk1 Indentation

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -7,4 +7,12 @@ def flash_class(level)
...
+  		return "#{attribute}" 
+  	else
+  		return " - "
```
Comment (godfreym) [inline app/helpers/application_helper.rb]: @leonardk1 check the whole attribute_present_or_not? method for indentation

Comment (leonardk1) [inline app/helpers/application_helper.rb]: @godfreym it has now been fixed

**Comment by godfreym** [comment]
@leonardk1 :+1:

**Comment by leonardk1** [comment]
@arafatm please review

**Comment by arafatm** [comment]
:+1:

**Pull request merged**